src/main/java/http/request/HttpRequest.java

```diff
@@ -0,0 +1,46 @@
+package http.request;
+
+import http.HttpBody;
+import http.HttpHeader;
+
+public class HttpRequest {
+    private HttpRequestLine requestLine;
+    private QueryString queryString;
+    private HttpHeader headers;
+    private HttpBody body;
```

Comment:
```suggestion
    private RequestHeader headers;
    private MessageBody body;
```
Http를 필드멤버에까지 넣지 않아도 될것 같습니다.!!

Reply:
음 그렇네요! 수정해야겠어요